### PR TITLE
Analysisprofile format price

### PR DESCRIPTION
**Pull request by Lunga001**
## Description

Show Price column on the AnalysisProfiles Listing

### CODE REVIEW

**Comment by luthando5** [comment]
It's working well except for this one section, the price column appears twice but as soon as you select it the other one disappears but then appears again after deselecting it.

image
image

File: src/bika/extras/browser/listingview/bika_analysisprofiles.py [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+             }
+        if not self.context.bika_setup.getShowPrices():
+            for i in range(len(self.listing.review_states)):
+                if "Price" in self.listing.review_states[i]["columns"]:
+                    self.listing.review_states[i]["columns"].remove("Price")
+            if "Price" in self.listing.columns:
+                del self.listing.columns['Price']
+            return
+
+        price = [
```
Comment (luthando5) [inline src/bika/extras/browser/listingview/bika_analysisprofiles.py:39]: 'Price' already exists in the listings column, so what you're doing here is essentially updating the toggle value from True to False. Just an insight,

Comment (Lunga001) [reply src/bika/extras/browser/listingview/bika_analysisprofiles.py:39]: No it doesn't, it's an override [from](https://github.com/senaite/senaite.core/blob/2.x/src/bika/lims/controlpanel/bika_analysisprofiles.py) and also depends on `self.context.bika_setup.getShowPrices()`

Comment (Lunga001) [reply src/bika/extras/browser/listingview/bika_analysisprofiles.py:39]: Setup > Accounting tab
image

**Pull request merged**